yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1075,7 +1090,7 @@ func addNotesToAssessmentReport() {
 	}
 }
 
-func addMigrationCaveatsToAssessmentReport(unsupportedDataTypesForLiveMigration []utils.TableColumnsDataTypes) {
+func addMigrationCaveatsToAssessmentReport(unsupportedDataTypesForLiveMigration []utils.TableColumnsDataTypes, unsupportedDataTypesForLiveMigrationWithFForFB []utils.TableColumnsDataTypes) {
```

Comment:
we should have constants for each feature name.
It will have to understand from the code - what all features we are reporting as unsupported.

Reply:
Yeah, we can have them for the feature name at all places. Maybe this and the above can go in a separate PR?

Reply:
yeah , will work on this just after this